build.gradle

```diff
@@ -219,11 +219,20 @@ subprojects {
     // Define a separate configuration for managing the dependency on Jetty ALPN agent.
     configurations {
         alpnagent
+
+        compile {
+            // Detect Maven Enforcer's dependencyConvergence failures. We only
+            // care for artifacts used as libraries by others.
+            if (!(project.name in ["grpc-benchmarks"])) {
+                //resolutionStrategy.failOnVersionConflict()
```

Comment:
Doh. Done.

When there's a failure you are sort of required to comment it out in order to debug where the failure is coming from. It's unfortunate.